packages/graphql-auth-transformer/src/ModelAuthTransformer.ts

```diff
@@ -478,6 +597,17 @@ Static group authorization should perform as expected.`
                 console.warn(
                     `It is not recommended to use 'queries' and 'operations'. The 'operations' argument will be used.`)
             }
+            if ( allow === 'groups' && (identityClaim || identityField)) {
+                throw new InvalidDirectiveError(`
+                @auth identifyField/Claim is only for 'allow: owner'`)
+            }
+            if (allow === 'owner' && groupClaim) {
+                throw new InvalidDirectiveError(`
+                @auth groupClaim is only for 'allow: groups'`);
+            }
+            if ( groupsField && groups) {
```

Comment:
I think this validation is incorrect, since ```groups``` defines the list of allowed groups, ```groupsField``` specifies which field to look for the group values in the type.